plugin/vite.config.location.ts

```diff
@@ -0,0 +1,115 @@
+/// <reference types="vitest" />
```

Comment:
This file isn't necessary ;)

Reply:
Done

Reply:
ca838bb63dff7bc9efdcc1e7ad0376fefc3c3404